anvil-rstudio-bioconductor/install.R

```diff
@@ -1,13 +0,0 @@
-# Install Bioconductor packages
```

Comment:
I think this file is referenced from here: https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-bioconductor/Dockerfile#L6

OK with removing it and putting the install commands in the above Dockerfile though.

Reply:
Thanks for catching this. I think maybe an option would be to just leave it in here for now.